data/model/oci/shared.py

```diff
@@ -1,16 +1,15 @@
-from data.database import Manifest, ManifestLegacyImage, Image
+from data.database import Manifest, ManifestSecurityStatus
 
 
-def get_legacy_image_for_manifest(manifest_id):
+def get_manifestsecuritystatus_for_manifest(manifest_id):
```

Comment:
nit: the word "manifest" is repeated in the function name, what about calling it something like `get_manifest_securitystatus` instead?

Reply:
It's the name of the actual table, so I'll just keep it as is to follow convention.